src/hpo_service.py

```diff
@@ -69,7 +72,8 @@ def startExperiment(self, name):
             started.release()
 
         if not value:
-            print("Starting experiment timed  out!")
+            logger.error(HPOErrorConstants.EXPERIMENT_TIMED_OUT)
+            return HPOErrorConstants.EXPERIMENT_TIMED_OUT
```

Comment:
Can we add a comment here on how the experiment can time out ?

Reply:
Added